services/slam/builtin/cartographer_int_test.go

```diff
@@ -5,15 +5,18 @@ import (
 	"io/ioutil"
 	"os"
 	"os/exec"
+	"path/filepath"
```

Comment:
Why are we changing integration tests if the PR description says that integration tests will come later? https://github.com/viamrobotics/rdk/pull/1776#issue-1537396492

Reply:
Talked offline but testing was needed to validate and it wasn't much work (replicating work done during the GetInternalState for carto) so I chose to include it here